app/code/core/Mage/Adminhtml/Block/Cms/Block/Widget/Chooser.php

```diff
@@ -41,7 +41,7 @@ class Mage_Adminhtml_Block_Cms_Block_Widget_Chooser extends Mage_Adminhtml_Block
     public function __construct($arguments=array())
     {
         parent::__construct($arguments);
-        $this->setDefaultSort('block_identifier');
+        $this->setDefaultSort('block_id');
```

Comment:
Iam sure this should be 'identifier'

Reply:
Clearly 'block_identifier' does not exist as a column in the database or in Magento source code . If we want to fix this issue in a first phase we can correct the error to 'block_id' which exists as column. 

If in time we decide to sort by default using the 'identifier' column we can change it but we should do some tests before. After it is merged I will review this in the future and I will give you my feedback between 'block_id' and 'title' I prefer the last one which is the easiest to find and understand. The block number or its identifier. It doesn't help me at all. I always click to sort by the meaning of the block.

Reply:
this is going onto the table `cms_block`, right?
so we have 
```

block_id | smallint(6) AI PK
title | varchar(255)
identifier | varchar(255)
content | mediumtext
creation_time | timestamp
update_time | timestamp
is_active | smallint(6)
```

Reply:
This is correct. In the future I will propose to do the default sorting by **title**. No one wants to see a table sorted by number or code.

Reply:
I would, in part because there is an existing index on identifier, but also because as a Developer I usually do not care about the title, only the identifier.
But maybe its better to make this generally configurable somehow.

Reply:
Not worth it. I'll come up with some screenshots when we have to determine which column the default sort is. I would say to correct only the error for the moment merging it and remaining at the block ID.